src/v1/destinations/iterable/strategies/track-identify.ts

```diff
@@ -0,0 +1,48 @@
+import { BaseStrategy } from './base';
+import { DestinationResponse, ResponseParams, Response } from '../types';
+import { checkIfEventIsAbortableAndExtractErrorMessage } from '../../../../v0/destinations/iterable/util';
+
+class TrackIdentifyStrategy extends BaseStrategy {
+  handleSuccess(responseParams: ResponseParams): {
+    status: number;
+    message: string;
+    destinationResponse: DestinationResponse;
+    response: Response[];
+  } {
+    const { destinationResponse, rudderJobMetadata, destinationRequest } = responseParams;
+    const { status } = destinationResponse;
+    const responseWithIndividualEvents: Response[] = [];
+
+    const { events, users } = destinationRequest?.body.JSON || {};
+    const finalData = events || users;
+
+    if (finalData) {
+      finalData.forEach((event, idx) => {
```

Comment:
use some type for the `event` -> which represents both `events` and `users` data to some extent

Reply:
we do not need to use the internal structure here anywhere, hence did not add it